src/modules/editor/sagas.ts

```diff
@@ -194,7 +194,7 @@ function* handleOpenEditor() {
   yield renderScene()
 
   // Enable snap to grid
-  yield put(toggleSnapToGrid(true))
+  yield handleToggleSnapToGrid(toggleSnapToGrid(true))
```

Comment:
I don't get this change, why are you avoiding the dispatch?

Reply:
To avoid calling all those mapStateToProps and mapDispatchToProps :)